app/views/expenses/index.html.erb

```diff
@@ -0,0 +1,68 @@
+<h1 class="m-2"> Expenditures List </h1>
+<div style="background-color: #e4ecf7">
+  <div class="d-flex flex-row-reverse bd-highlight" style="background-color: #3778c2;">
+    <div class="p-3 bd-highlight" style="color: #9cb8db">Most ancient</div>
+    <div class="p-3 bd-highlight" style="color: #c1d2e8">Most recent</div>
+  </div>
+
+  <div class="d-flex flex-row-reverse bd-highlight bg-white">
+    <div class="p-2 bd-highlight">$00.00</div>
+    <div class="p-2 bd-highlight">Gift credit</div>
+  </div>
+  <br/>
+  
+  <div class="d-flex align-items-center">
+    <% if @group.present? %>
+        <div class="pb-2" style="margin-left: 0.5rem;">
+          <%= raw @group.icon %>
+        </div>
+        <div class="ms-3">
+          <h5 style="font-weight: bold;"><%= @group.name %></h5>
+        </div>
+      <% end %>
+  </div>
+
+  <div class="table-responsive">
+    <table class="table table-bordered table-striped">
+      <thead>
+        <tr>
+          <th class="col-4">Name</th>
+          <th class="col-4">Amount</th>
+          <th class="col-4">Created At</th>
+        </tr>
+      </thead>
+      <tbody>
+        <% @expenses.each do |expense| %>
+          <tr>
+            <td class="col-4"><%= expense.name %></td>
+            <td class="col-4"><%= number_to_currency(expense.amount) %></td>
+            <td class="col-4"><%= expense.created_at %></td>
+          </tr>
+        <% end %>
+      </tbody>
+    </table>
+  </div>
+  </br>
+
+  <% if @group.present? %>
+    <div class="row">
+      <div class="col-6">
+        <h6 style="margin-left: 0.5rem; font-weight: bold;">TOTAL AMOUNT:</h6>
+      </div>
+      <div class="col-6 text-end">
+        <p style="margin-right: 2rem; font-weight: bold;"><%= number_to_currency(@total_amount) %></p>
+      </div>
+    </div>
+  <% end %>
+
+  <% if @group.present? %>
+    <div class="p-2 d-flex justify-content-between" style="background-color: #5fb523; position: fixed; bottom: 0; width: 100vw;">
+      <div style="margin-right: 2rem">
+        <%= link_to "Add expense", new_group_expense_path(@group), class: "btn btn-outline-light" %>
+      </div>
+      <div>
+        <%= link_to "Back to groups", groups_path, class: "btn btn-outline-light" %>
+      </div>
+    </div>
+  <% end %>
+</div>
```

Comment:
- [ ] I understand that programmers love being creative with designs. However, it is paramount that designers stick to design guidelines in projects. This shows utmost professionalism and attention to detail. On this page, I would recommend you do the following ✍️,

- Move the **_Back to groups button_** to the top of the page in the navigation section (You don't need to change it to an icon, just position it at the right corner of the navigation section. 😃 
- Make the _**Add expense**_ look identical to that of the Behance design layout for buttons on their templates. 🖥️  

--- 

| Your Design  | Behance Design |
| ------------- | ------------- |
| ![image](https://github.com/DevaWinner/expenditure/assets/77106962/056ab3d9-eeab-4f5c-ad64-ccc414333ca9)  | ![image](https://github.com/DevaWinner/expenditure/assets/77106962/e7ad23e4-29d8-42f7-8551-8d0be3820389)  |

Reply:
Fixed.